R/getBaselineDivergence.R

```diff
@@ -6,9 +6,20 @@
 #' The method operates on `SummarizedExperiment` objects, and the results
 #' are stored in `colData`.
 #'
-#' @inheritParams getStepwiseDivergence
-#' @param baseline_sample \code{Character vector}. Specifies the baseline sample(s) to be used. If the
-#'   "group" argument is given, this must be a named vector; one element per group. (Default: \code{NULL})
+#' @inheritParams addStepwiseDivergence
+#' @param FUN \code{Function} for dissimilarity calculation. The function must
+#' expect the input matrix as its first argument. With rows as samples and 
+#' columns as features. (Default: \code{vegan::vegdist})
+#' @param altexp \code{Character scalar} or \code{integer scalar}. Specifies the 
+#' alternative experiment containing the input data. (Default: \code{NULL})
```

Comment:
This might be good to move as hidden parameter as this is not core functionality